src/news/newsActions.ts

```diff
@@ -0,0 +1,17 @@
+import { Action, ActionCreator } from 'redux';
+import { ThunkAction } from 'redux-thunk';
+import { API, GlobalState } from '../types'
+import { News } from './types'
+
+export interface NewsAction extends Action { type: typeof NEWS_RECEIVE_ACTION, newsItems: News[] }
```

Comment:
To be consistent with the rest of the code, maybe rename to `NewsReceivedAction`